Documentation/tutorials/EdgeBridgeTutorialAppFinal/EdgeBridgeTutorialApp/AppDelegate.swift

```diff
@@ -13,13 +13,12 @@
 import AEPAssurance
 import AEPCore
 import AEPEdge
-import AEPEdgeConsent
 import AEPEdgeBridge
+import AEPEdgeConsent
 import AEPEdgeIdentity
 import AEPLifecycle
 
-//* EdgeBridge Tutorial - code section (1/2)
-import AEPEdgeBridge
+// * EdgeBridge Tutorial - code section (1/2)
 // EdgeBridge Tutorial - code section (1/2) */
```

Comment:
this got reordered after formatting, but we should keep the comment around the import statement for AEPEdgeBridge